<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>480</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Создать путь</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>160</width>
     <height>221</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Долгота</string>
      </property>
      <property name="buddy">
       <cstring>longitude</cstring>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="longitude">
      <property name="text">
       <string>0.0</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Широта</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="latitude">
      <property name="text">
       <string>0.0</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Высота</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="altitude">
      <property name="text">
       <string>0.0</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="add_pointButton">
      <property name="text">
       <string>&amp;Добавить точку</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="calcButton">
      <property name="text">
       <string>&amp;Рассчитать</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>10</y>
     <width>161</width>
     <height>192</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLineEdit" name="focus">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="ver_matrix"/>
    </item>
    <item>
     <widget class="QLineEdit" name="hor_matrix"/>
    </item>
    <item>
     <widget class="QLineEdit" name="distance"/>
    </item>
    <item>
     <widget class="QLineEdit" name="ver_overlap"/>
    </item>
    <item>
     <widget class="QLineEdit" name="hor_overlap"/>
    </item>
    <item>
     <widget class="QLineEdit" name="build_high"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>165</width>
     <height>191</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Фокус, мм</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Высота матрицы, мм&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Ширина матрицы, мм&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>Дистанция до объекта</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Перекрытие по вертикали, %</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Перекрытие по горизонтали, %</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>Высота здания, м</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="settingsButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>163</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Принять настройки</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>250</y>
     <width>161</width>
     <height>221</height>
    </rect>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>focus</tabstop>
  <tabstop>ver_matrix</tabstop>
  <tabstop>hor_matrix</tabstop>
  <tabstop>distance</tabstop>
  <tabstop>ver_overlap</tabstop>
  <tabstop>hor_overlap</tabstop>
  <tabstop>build_high</tabstop>
  <tabstop>settingsButton</tabstop>
  <tabstop>longitude</tabstop>
  <tabstop>latitude</tabstop>
  <tabstop>altitude</tabstop>
  <tabstop>add_pointButton</tabstop>
  <tabstop>calcButton</tabstop>
  <tabstop>textBrowser</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
